src/plugins/plugins.js

```diff
@@ -58,6 +60,8 @@ define([
 
     plugins.ImportExport = ImportExport;
 
+    plugins.ImportedRootPlugin = ImportedRootPlugin;
+
```

Comment:
Would also like to see some documentation in API.md for this, unless you're intentionally deferring that for maturation or something

Reply:
Waiting for maturation.  Think this is not fully stable yet.